ultraScurve.py

```diff
@@ -51,13 +51,11 @@
 
 # Setup the output TTree
 from treeStructure import gemTreeStructure
-gemData = gemTreeStructure('scurveTree','Tree Holding CMS GEM SCurve Data')
+gemData = gemTreeStructure('scurveTree','Tree Holding CMS GEM SCurve Data',scanmode.SCURVE)
+gemData.setDefaults(options)
 gemData.calPhase[0] = options.CalPhase
 gemData.l1aTime[0] = options.L1Atime
 gemData.latency[0] = options.latency
```

Comment:
Probably want to pull these out of the chip a la ultraLatency.py#115 (general comment for chip parameters that are written to the tree)

Reply:
I have implemented your suggestion.  Now our scan scripts will read relevant parameters from the chip to store them.